packages/app-accounts/src/UploadButton.tsx

```diff
@@ -0,0 +1,205 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { BareProps, I18nProps } from '@polkadot/ui-app/types';
+import { KeyringPair$Json } from '@polkadot/util-keyring/types';
+
+import React from 'react';
+
+import { InputAddress } from '@polkadot/ui-app/InputAddress';
+import isUndefined from '@polkadot/util/is/undefined';
+import arrayContainsArray from '@polkadot/ui-app/util/arrayContainsArray';
+import File from '@polkadot/ui-app/Params/Param/File';
+import keyring from '@polkadot/ui-keyring/index';
+
+import UploadModal from './UploadModal';
+import translate from './translate';
+
+let actualJsonProperties: Array<string> = [];
```

Comment:
Why are there not in the class? They are supposed to be singletons? (i.e. exist and have value across instances?)